guide-redaction/index.md

```diff
@@ -234,38 +234,40 @@ title: "Guide de rédaction du contenu du site Canada.ca"
+    <div class="row mrgn-tp-lg mrgn-bttm-md">
+      <div class="col-md-8">
+        <figure><img alt="Pyramide inversée illustrant comment créer une page Web en indiquant tout d'abord les renseignements clés." class="img-responsive" id="fig1" src="/guide-redaction/images/pyramide-fr.png" />
+          <figcaption>
+            <p><strong>Figure 1:</strong> pyramide inversée pour l’organisation du contenu</p>
+          </figcaption>
+        </figure>
+      </div>
+    </div>
+    <p>La pyramide inversée est divisée en quatre parties. Le contenu le plus important pour la réalisation des tâches se trouve au sommet, la plus grosse partie de la pyramide. La pyramide inversée représente le degré décroissant de pertinence du contenu pour la réalisation des tâches au fur et à mesure que l’on se rapproche du sommet (au bas).</p>
+    <ol>
+      <li><strong>Information la plus importante</strong>&nbsp;:
         <ul>
-          <li>Premièrement&nbsp;:
-            <ul>
-              <li>commencer chaque page Web par l'idée, l'étape ou l'information la plus importante.</li>
-            </ul>
-          </li>
-          <li>Deuxièmement&nbsp;:
-            <ul>
-              <li>ajouter les détails et les tâches par ordre d'importance;</li>
-              <li>mettre tout le contenu lié à la tâche au centre de la page;</li>
-              <li>utiliser des liens pour orienter les gens vers les renseignements et les tâches essentielles.</li>
-            </ul>
-          </li>
-          <li>Troisièmement&nbsp;:
-            <ul>
-              <li>inclure des renseignements pertinents à l'intention des publics cibles qui désirent en apprendre davantage ou qui ont une connaissance approfondie du sujet ou du service.</li>
-            </ul>
-          </li>
-          <li>Quatrièmement&nbsp;:
-            <ul>
-              <li>ajouter des liens seulement s'ils concernent le contenu et appuient la tâche.</li>
-            </ul>
-          </li>
+          <li>commencer chaque page Web par l'idée, l'étape ou l'information la plus importante.</li>
         </ul>
-      </details>
-    </figure>
+      </li>
+      <li><strong>Tâches prioritaires</strong>&nbsp;:
+        <ul>
+          <li>ajouter les détails et les tâches par ordre d'importance;</li>
+          <li>mettre tout le contenu lié à la tâche au centre de la page;</li>
+          <li>utiliser des liens pour orienter les gens vers les renseignements et les tâches essentielles.</li>
+        </ul>
+      </li>
+      <li><strong>Plus d'information</strong>&nbsp;:
+        <ul>
+          <li>inclure des renseignements pertinents à l'intention des publics cibles qui désirent en apprendre davantage ou qui ont une connaissance approfondie du sujet ou du service.</li>
+        </ul>
+      </li>
+      <li><strong>Liens connexes</strong>&nbsp;:
+        <ul>
+          <li>ajouter des liens seulement s'ils concernent le contenu et appuient la tâche.</li>
```

Comment:
This is missing the parentheses and info inside it at the end, the full bullet point should be: 

ajouter des liens seulement s'ils concernent le contenu et appuient la tâche (par exemple, des lois ou des publications).

Reply:
Done